### ISSUE TITLE
Document that Paging component paginglabel should be unique and descriptive

### ISSUE THREAD

**Issue by Blackbaud-ToddRoberts** [blackbaud/skyux2-docs#281]
From #1266 

Update the paging documentation to note that values for `paginglabel` property should be unique and descriptive so that users of screen readers can know what they are navigating.

**Comment by blackbaud-johnly** [comment]
Addressed in https://github.com/blackbaud/skyux2-docs/pull/312.
